setup-router/src/components/Navbar.js

```diff
@@ -1,14 +1,21 @@
-import React, { Component } from 'react';
+import React, { Component } from "react";
+import { NavLink } from "react-router-dom";
 
-import '../styles/App.css';
+import "../styles/App.css";
 
 export default class Navbar extends Component {
   render() {
     return (
       <ul className="nav">
-        <li className="active">Home</li>
-        <li>About Us</li>
-        <li>Contact</li>
+        <li>
```

Comment:
@prateekinstillion  to further simplify here, based on my own research, I found that  `react-router` provided a `Switch` component itself. Here's the link 
https://reacttraining.com/react-router/web/api/Switch

It might clear up the HTML a bit.

Reply:
Just forgot that to add. Thanks @abhi18av. I will fix that.

Reply:
Most welcome :)